CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+![ci workflow status](https://img.shields.io/github/actions/workflow/status/apollographql/apollo-mcp-server/ci.yml)
+![release binaries workflow status](https://img.shields.io/github/actions/workflow/status/apollographql/apollo-mcp-server/release-bins.yml?label=release%20binaries)
+![release container workflow status](https://img.shields.io/github/actions/workflow/status/apollographql/apollo-mcp-server/release-container.yml?label=release%20container)
+![version](https://img.shields.io/github/v/release/apollographql/apollo-mcp-server)
+![license](https://img.shields.io/github/license/apollographql/apollo-mcp-server)
+
+## How to contribute to Apollo MCP Server
+
+#### Bug Reporting
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/apollographql/apollo-mcp-server/issues) as well as the [Apollo Community forums](https://community.apollographql.com/latest).
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/apollographql/apollo-mcp-server/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+* If appropriate add the most relevant label but leave empty if unsure.
+
+#### **Did you write a patch that fixes a bug?**
+
+* Refer to the simple branching guide for the project.
+* Open a new GitHub pull request with the patch.
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+* Before submitting, please read the [Contributing to Apollo MCP Server](#contributing-to-apollo-mcp-server) guide to know more about coding conventions, branching strategies, etc.
```

Comment:
The text "to know more about ..." reads weird to me. Perhaps we could have something like

```suggestion
* Before submitting, please read the [Contributing to Apollo MCP Server](#contributing-to-apollo-mcp-server) guide to learn more about our coding conventions, branching strategies, etc.
```